example/cds-hooks-api-guide/src/suggestions-links-fhir/data.js

```diff
@@ -0,0 +1,96 @@
+export const suggestionData = {
+  bloodThinner: {
+    label: "Create a prescription for Rivaroxaban (Xarelto) 10 MG",
+    actions: [
+      {
+        type: "create",
+        description: "Create a prescription for Rivaroxaban (Xarelto) 10 MG",
+        resource: {
+          resourceType: "MedicationRequest",
+          id: "3ba900b2-a795-40a0-8aae-1cfbb02e3ac1",
+          status: "active",
+          intent: "order",
+          medicationCodeableConcept: {
+            coding: [
+              {
+                system: "http://www.nlm.nih.gov/research/umls/rxnorm",
+                code: "429503",
+                display: "Rivaroxaban 10 MG",
+              },
+            ],
+            text: "Rivaroxaban 10 MG",
+          },
+          subject: {
+            reference: "urn:uuid:b626136e-aff8-4711-8279-536f07f197b5",
```

Comment:
Depending on whether or not you want the ID to be a static thing (in some cases you definitely do I think - so that it's easier to set up), you could also have 

`urn:uuid:${randomUUID}`